modular/manager/bucket_migrate_scheduler.go

```diff
@@ -459,18 +464,21 @@ func (s *BucketMigrateScheduler) processEvents(migrateBucketEvents *types.ListMi
 
 func (s *BucketMigrateScheduler) subscribeEvents() {
 	go func() {
-		UpdateBucketMigrateSubscribeProgressFunc := func() {
+		logNumber := uint64(0)
+
+		UpdateBucketMigrateSubscribeProgressFunc := func(idx uint64) {
```

Comment:
There is no need to pass parameters here, just use `logNumber` directly. And also `idx` is not a good parameter name.

Reply:
The argument is a counter/number, not an index. idx represents the index normally.